draft-ietf-httpbis-resumable-upload.md

```diff
@@ -577,38 +577,38 @@ While the request content is being received, the server SHOULD send interim resp
 
 ### Examples {#upload-appending-example}
 
-A) The following example shows an upload append request. The client transfers the next 25000000 bytes at an offset of 25000000 and does not indicate that the upload is then completed. The server generates one interim response and finally acknowledges the new offset:
+A) The following example shows an upload append request. The client transfers the next 23456789 bytes at an offset of 23456789 and does not indicate that the upload is then completed. The server generates one interim response and finally acknowledges the new offset:
 
 ~~~ http-message
 PATCH /upload/b530ce8ff HTTP/1.1
 Host: example.com
 Upload-Complete: ?0
-Upload-Offset: 25000000
-Content-Length: 25000000
+Upload-Offset: 23456789
+Content-Length: 23456789
 Content-Type: application/partial-upload
 
-[content (25000000 bytes)]
+[content (23456789 bytes)]
 ~~~
 
 ~~~ http-message
 HTTP/1.1 104 Upload Resumption Supported
-Upload-Offset: 12500000
+Upload-Offset: 35185184
 
 HTTP/1.1 204 No Content
 Upload-Complete: ?0
 ~~~
 
-B) The next example shows an upload append, where the client transfers the remaining 25000000 bytes and completes the upload. The server processes the uploaded representation and generates the responding response, in this example containing extracted meta data:
+B) The next example shows an upload append, where the client transfers the remaining 23456789 bytes and completes the upload. The server processes the uploaded representation and generates the responding response, in this example containing extracted meta data:
 
 ~~~ http-message
 PATCH /upload/b530ce8ff HTTP/1.1
 Host: example.com
 Upload-Complete: ?1
-Upload-Offset: 25000000
-Content-Length: 25000000
+Upload-Offset: 46913578
```

Comment:
Should we make sure that the final completed length is 1234567890? So each chunk can be 1200000000, 34000000, 560000, 7800, 90?

Reply:
My understanding of the examples was that they are fairly isolated from each other. Even prior to the change to use larger numbers, in https://www.ietf.org/archive/id/draft-ietf-httpbis-resumable-upload-09.html#section-4.4.3 the appends result in an object that has a total length of 300 bytes, which doesn't correlate to any upload length in other examples.

Furthermore, I'd prefer to keep examples concise. Adding multiple append operations that don't do much beyond using a different size increases the length of example and might add more source of confusion. I created https://github.com/httpwg/http-extensions/issues/3250 that highlights this via reuse of the same URL

Reply:
Oh I wasn't suggesting adding more appends to the example, I was just thinking 46913578 kind of loses the benefit of the easily recognizable number. If offset is 1230000000 and length is 4567890, it's obvious that they add up together as 1234567890

Reply:
Ah I undestand now, I can do that